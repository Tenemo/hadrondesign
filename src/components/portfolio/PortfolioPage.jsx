import React from 'react';

const PortfolioPage = () => {
        return (
        <div className="jumbotron">
            <h1>Portfolio</h1>
        </div>
    );
};

export default PortfolioPage;
