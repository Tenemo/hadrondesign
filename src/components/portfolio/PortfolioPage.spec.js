import React from 'react';
import PortfolioPage from './PortfolioPage';

describe('PortfolioPage', () => {
    it('matches snapshot', () => {
        const wrapper = shallow(
            <PortfolioPage />
        );
        expect(wrapper).toMatchSnapshot();
    });
});