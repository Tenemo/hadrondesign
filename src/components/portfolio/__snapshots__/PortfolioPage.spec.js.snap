// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PortfolioPage matches snapshot 1`] = `
<div
  className="jumbotron"
>
  <h1>
    Portfolio
  </h1>
</div>
`;
